<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSerial</class>
 <widget class="QWidget" name="FormSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="btnSend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxConfig">
     <property name="title">
      <string>Config</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <widget class="QToolButton" name="tBtnRefresh">
        <property name="text">
         <string>refresh</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxPort"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelBaudRate">
          <property name="text">
           <string>Baud Rate</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBoxBaudRate"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelDataBit">
          <property name="text">
           <string>Data Bit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBoxDataBit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelCheckBit">
          <property name="text">
           <string>Check Bit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="comboBoxCheckBit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelStopBit">
          <property name="text">
           <string>Stop Bit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="comboBoxStopBit"/>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btnSerialSwitch">
        <property name="text">
         <string>Open/Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxRecv">
     <property name="title">
      <string>Receive</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QTextBrowser" name="textBrowserRecv"/>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelRecvCount">
          <property name="text">
           <string>RecvCount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnIntegration">
          <property name="text">
           <string>integration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tBtnData">
          <property name="text">
           <string>data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxSend">
     <property name="title">
      <string>Send</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabSingle">
         <attribute name="title">
          <string>Single</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTextEdit" name="textEditSend"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabMultipe">
         <attribute name="title">
          <string>Multipe</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="2,8">
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="hLay3">
            <item>
             <widget class="QLineEdit" name="lineEditCmd3"/>
            </item>
            <item>
             <widget class="QToolButton" name="tBtn3">
              <property name="text">
               <string>3</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="lineEditPrompt3"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEditPrompt1"/>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="lineEditPrompt2"/>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="hLay4">
            <item>
             <widget class="QLineEdit" name="lineEditCmd4"/>
            </item>
            <item>
             <widget class="QToolButton" name="tBtn4">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEditPrompt4"/>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="lineEditPrompt0"/>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="hLay0">
            <item>
             <widget class="QLineEdit" name="lineEditCmd0"/>
            </item>
            <item>
             <widget class="QToolButton" name="tBtn0">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="hLay1">
            <item>
             <widget class="QLineEdit" name="lineEditCmd1"/>
            </item>
            <item>
             <widget class="QToolButton" name="tBtn1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="hLay2">
            <item>
             <widget class="QLineEdit" name="lineEditCmd2"/>
            </item>
            <item>
             <widget class="QToolButton" name="tBtn2">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Serial</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
